files/build_templates/sonic_debian_extension.j2

```diff
@@ -111,6 +111,9 @@ sudo bash -c "echo enabled=false > $FILESYSTEM_ROOT/etc/sonic/updategraph.conf"
 # Copy SNMP configuration files
 sudo cp $IMAGE_CONFIGS/snmp/snmp.yml $FILESYSTEM_ROOT/etc/sonic/
 
+# Copy initial DHCP relay configuration file, will be overwritten after docker starts
```

Comment:
No I don't think anyone will overwrite this file. This file is the input of config.sh for dhcp relay docker and the real configuration will be generated according to this file.